tracer/build/_build/Build.cs

```diff
@@ -178,6 +178,7 @@ void DeleteReparsePoints(string path)
         .Description("Builds NuGet packages, MSIs, and zip files, from already built source")
         .After(Clean, BuildTracerHome, BuildProfilerHome, BuildNativeLoader)
         .DependsOn(CreateRequiredDirectories)
+        .DependsOn(CopyDdDotnet)
         .DependsOn(ZipMonitoringHome)
         .DependsOn(BuildMsi)
```

Comment:
You probably want `DependsOn(CopyDdDotnet)` in `ZipMonitoringHome` and `BuildMsi` rather than here I think (to ensure correct ordering)